### PR TITLE
update removeTask in project

### PR DESCRIPTION
**Pull request by Mihail-Ch**
Good

### CODE REVIEW

**Review by Mihail-Ch — COMMENTED** [review]
Отлично!

**Review by MichailChertov — APPROVED** [review]
I finish work

**Review by Mihail-Ch — COMMENTED** [review]
Ok

**Review by Mihail-Ch — COMMENTED** [review]
OK

**Review by MichailChertov — APPROVED** [review]
look

**Comment by MichailChertov** [comment]
good

File: .DS_Store
Comment (Mihail-Ch) [inline .DS_Store:1]: Ok

File: UserDefoults.xcodeproj/project.xcworkspace/xcuserdata/mihailchertov.xcuserdatad/UserInterfaceState.xcuserstate
Comment (Mihail-Ch) [inline UserDefoults.xcodeproj/project.xcworkspace/xcuserdata/mihailchertov.xcuserdatad/UserInterfaceState.xcuserstate:1]: Ok

File: UserDefoults/Controller/Delegate.swift
Comment (Mihail-Ch) [inline UserDefoults/Controller/Delegate.swift:1]: Ok

File: .DS_Store
Comment (MichailChertov) [inline .DS_Store:1]: write

File: UserDefoults.xcodeproj/project.xcworkspace/xcuserdata/mihailchertov.xcuserdatad/UserInterfaceState.xcuserstate
Comment (MichailChertov) [inline UserDefoults.xcodeproj/project.xcworkspace/xcuserdata/mihailchertov.xcuserdatad/UserInterfaceState.xcuserstate:1]: write

File: UserDefoults/Controller/Delegate.swift
Comment (MichailChertov) [inline UserDefoults/Controller/Delegate.swift:1]: write

**Review by MichailChertov — APPROVED** [review]

**Comment by MichailChertov** [comment]
look

**Comment by Mihail-Ch** [comment]
finish

**Pull request closed without merging**